functions/go/enable-gcp-services/gcpservices/project_service_list.go

```diff
@@ -0,0 +1,230 @@
+package gcpservices
```

Comment:
Do we expect others to import this package?
If not, we can keep it flat and we will not need to define `type Result struct` to pass the information around. Instead we can directly operate on `ResultItem`.

Reply:
Initially I thought we would reuse for the generate-services fn, but switched to ResultItem since that was dropped.